src/data_provider/tests/sysInfoPackagesMAC/pkgWrapper_test.cpp

```diff
@@ -42,8 +42,8 @@ TEST_F(PKGWrapperTest, LongVersion)
     EXPECT_EQ(wrapper->architecture(), UNKNOWN_VALUE);
     EXPECT_EQ(wrapper->format(), "pkg");
     EXPECT_EQ(wrapper->osPatch(), "");
-    EXPECT_EQ(wrapper->source(), "utilities");
-    EXPECT_EQ(wrapper->location(), inputPath + "/" + package + "/" + APP_INFO_PATH);
+    EXPECT_EQ(wrapper->source(), "applications");
```

Comment:
Just a question: why `utilities` became `applications` ?

Reply:
This is because in the previous pkgWrapper, there was a bug not detected that causes that all locations were considered as utilities.
![image](https://github.com/wazuh/wazuh/assets/101227434/3d1282b2-f1f3-4902-9a66-9518606dba10)
This was fixed as: 
![image](https://github.com/wazuh/wazuh/assets/101227434/84e13559-ba14-4780-afed-fe9cdeb61730)
Because of this, the test should be also fixed.

Reply:
Good catch!